docs/pages/zero-trust-access/deploy-a-cluster/multi-region-blueprint.mdx

```diff
@@ -25,6 +25,10 @@ Before continuing, you must acknowledge the following warnings:
 - You are responsible for deploying and maintaining the Teleport dependencies
   including the database, object storage, container runtime, network plugins
   and anything else which is required.
+- Regions must be provisioned with enough capacity to handle all traffic in
+  the event of a complete region failure. This includes the ability to
+  horizontally scale Teleport components and immediately absorb new
+  traffic as nodes and users reconnect to the healthy regions.
```

Comment:
> This includes the ability to
  horizontally scale Teleport components and immediately absorb new
  traffic as nodes and users reconnect to the healthy regions.

This looks like we are telling them to autoscale, which is not supported and a bad idea. I'd just remove this bot, the original part seems enough.

Reply:
I removed the horizontal autoscaling.
I wonder if it's bad for an emergency.  It's not ideal for a normal operation as it causes too many disruptions but for an emergency I don't see why it would be bad tho

Reply:
Pretty sure users would mess it up and create some HPA that breaks connections. Tuning HPAs properly is very complex and I'd rather have them scale up manually or provision more capacity than build a rube goldberg machine they don't understand and that won't work the day you have an outage.